osversion_windows.go

```diff
@@ -82,6 +89,13 @@ var specificVersions = map[string]string{
 	"6.3.9200":   "Windows 8.1 / Windows Server 2012 R2",
 	"6.3.9600":   "Windows 8.1, Update 1",
 	"10.0.10240": "Windows 10 RTM",
+	"10.0.10586": "Windows 10 1511 / Windows Server 2016 Technical Preview 4",
```

Comment:
Per https://www.windows-commandline.com/find-windows-os-version-from-command/, there are several ways to get human readable string directly. `systeminfo` is rather slow, but I found `wmic os get Caption,CSDVersion` works great across versions.

```
Caption                            CSDVersion
Microsoft Windows XP Professional  Service Pack 3
```

```
Caption                      CSDVersion
Microsoft Windows 10 家庭版
```
The benefit is that we don't need to maintain the mapping any more.